client/src/utils/storage.js

```diff
@@ -1,19 +1,26 @@
 const storage = {
-  get: (key) => {
-    const inLS = localStorage.getItem(key);
-
-    if (inLS) {
-      return JSON.parse(inLS);
+  get: (key, session = false) => {
+    const result = session ? sessionStorage.getItem(key) : localStorage.getItem(key);
+    if (!result) {
+      return null;
     }
 
-    return null;
+    return JSON.parse(result);
+  },
+  set: (key, value, session = false) => {
+    if (session) {
+      sessionStorage.setItem(key, JSON.stringify(value));
+    } else {
+      localStorage.setItem(key, JSON.stringify(value));
+    }
   },
-  set: (key, value) => {
-    localStorage.setItem(key, JSON.stringify(value));
+  remove: (key, session = false) => {
+    if (session) {
+      sessionStorage.removeItem(key);
+    } else {
+      localStorage.removeItem(key);
+    }
   },
-  remove: (key) => {
-    localStorage.removeItem(key);
-  }
 };
 
```

Comment:
It looks like a lot of duplicate code. How about exporting two separate objects?
Also, we should be able to fetch value just by `key` but it's requiring additional param i.e. `session`.
 
```
const storageProvider = (storage) => {
    
}
export const SS = storageProvider(window.sessionStorage);
export const LS = storageProvider(window.localStorage);
```

Reply:
Done, I applied the change to all the files that required the storage utility.